### ISSUE TITLE
.rand seed count repeat does not offset properly?

### ISSUE THREAD

**Issue by HnimNart** [kirkedal/compSys-e2017-sim#23]
Using  e.g .rand  1234 4 will only offset the next position by 8 bytes and not by 4 * 8 bytes. 
Running the below code from the handed out matrixmult-big.x64:
```

    .pos 0x400
stack:
dim:
    .quad 2
mat1:
    .rand 1234 4
mat2:
    .rand 1234 4
out:
```

We end up writing to _out_:
```
  [0x003F8]                0 ->               35
  [0x00418]            37377 ->       3874743027
  [0x00420]            19373 ->  185540953749992
  [0x00428]            47884 -> -99301950626761295
  [0x00430]            57611 -> 611121257137996817

```
My intuition is that we should start writing from addr. 0x00448 instead
Is this intended behaviour?

Br

**Comment by HnimNart** [comment]
Hi, you can surpass it by manually setting a position for the `out` address. I've set it to 0x10000, which then makes sure you don't overwrite any elements when the max dimension is 50.

**Comment by finnschiermer** [comment]
Should be fixed by #25
